bin/node/src/service.rs

```diff
@@ -326,7 +326,7 @@ pub fn new_authority(
     let backoff_authoring_blocks = Some(LimitNonfinalized(aleph_config.max_nonfinalized_blocks()));
     let prometheus_registry = config.prometheus_registry().cloned();
 
-    let import_queue_handle = BlockImporter(import_queue.service());
+    let import_queue_handle = BlockImporter::new(import_queue.service(), metrics.clone());
```

Comment:
We can move this to `finality-aleph`, but it would require passing `import_queue.service()` (an entity from substrate's world) in `AlephConfig`, which I don't quite like. Alternatively, we can contruct `BlockImporter` there without metrics, and attach them in `finality-aleph`